person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, name: 'Unknown', age: 1, parent_permission: true)
+    @id = id
```

Comment:
- [ ] Good job using getters and setters on your variables. However, as required for the project to have default values of `"Unknown"` and `true` on `name` and `parent_permission`, I think it would be better if you you update your `initialize` method to match the requirement. And Optionally you can use the `Random` method on your `id` variable so it can help you to generate `id` automatically. I suggest to change it and look similar to this:
```ruby
  def initialize(age, name = "Unknown", parent_permission: true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```